site/proxy.md

```diff
@@ -45,8 +45,9 @@ argument, e.g.
 
     host1$ weave launch-proxy -H tcp://127.0.0.1:9999
 
-When launching the proxy via TLS, `-H` and/or [TLS options](#tls) are
-required.
+If no TLS or listening interfaces are set, TLS will be autoconfigured
+based on the docker daemon's settings, and listening interfaces will
+be autoconfigured based on your docker client's settings.
```

Comment:
So you could end up with TLS being configured but listening on a domain socket, which presumably doesn't support TLS?